tools/deploy-qemu.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
A shebang would be nice.
```
#!/usr/bin/env python3
```

Reply:
Sure, but which one?!

FTR: we agreed `/usr/bin/env` is good enough for this kind of script.